### ISSUE TITLE
RubyGems not packed with assets?

### ISSUE THREAD

**Issue by sleungcy** [madetech/unveil-rails#2]
I downloaded the gem from rubygems, and it doesn't seem to contain the assets directory? The install tasks isn't working without the default blank.gif

vagrant@precise64:/mnt/mycode$ bin/rails g unveil:rails:install
Warning: Running `gem pristine --all` to regenerate your installed gemspecs (and deleting then reinstalling your bundle if you use bundle --path) will improve the startup performance of Spring.
 initializer  unveil.rb
Could not find "blank.gif" in any of your source paths. Your current source paths are:
/mnt/mycode/vendor/ruby/2.2.0/gems/unveil-rails-0.2.1/lib/unveil/assets

**Comment by lukemorton** [comment]
Ah you're correct. Let me fix that and push a new release.

**Comment by lukemorton** [comment]
Could you try this change in your Gemfile:

``` ruby
gem 'unveil-rails', github: 'madetech/unveil-rails', branch: 'fix-2'
```

If it's good I'll bundle a new release.

**Comment by dinjas** [comment]
I hit the same issue and the fix solved my problem. Thanks

**Comment by lukemorton** [comment]
I'll get this merged into master and a new version released tomorrow. Thanks.

**Comment by lukemorton** [comment]
I have released a new version of the gem, `0.2.2`. You can now remove the github specifics from your `Gemfile`'s:

``` ruby
gem 'unveil-rails'
```
